<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgValidTimePeriodEdit</class>
 <widget class="QDialog" name="dlgValidTimePeriodEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbTimePeriod">
     <property name="title">
      <string> Valid time period </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblFrom">
        <property name="text">
         <string>From :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="teFrom">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTo">
        <property name="text">
         <string>To :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="teTo">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btbButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>dlgValidTimePeriodEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>dlgValidTimePeriodEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
